### ISSUE TITLE
Events room fridge is broken

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1234]
When I came in this morning there was a smell of magic smoke in the main room. I happened to go into the events room and noticed that the door seemed to be unlocked and the smell was much stronger there.

I checked the fuses and it seems the sockets have gone off, I turned them back on and they went on fine, the fridge seemed to turn on and the doors did lock, but after a few minutes the door bot hadn't come back up so I powered down everything for now as I haven't got time to investigate yet.

**Comment by johnmckerrell** [comment]
I turned the power back on and was opening the box to the door bot to see if something had gone wrong and a bang came from the fridge in the events room and the sockets tripped again. With that off the door bot worked fine, just took a short while to come back up. I've wrapped the fridge plug in insulation tape to dissuade plugging it back in until someone can take a look.

**Comment by DoESsean** [comment]
Can we get this looked at at the next Repair Cafe? Seems like it would be quite a good 'big' project that could make everyone cheer again :)

**Comment by amcewen** [comment]
It would be good to get this fixed (or sort out some sort of refrigeration in the events room) as it's tricky having all the drinks in the main room for events.

**Comment by chris-does** [comment]
After the Repair Café had finished today, I started to take a look at this. . .

### TLDR
I’m making some progress and, due to there being an external event booked in the Events Space all day tomorrow (Monday 28th October), I have removed the fridge from the Events Space and tried to put it out of the way near where I tend to sit (in the corner of Room 29) so that I can continue working on it tomorrow.

N.B. The fridge is currently **not fully assembled** and is also on a wheeled trolley which is significantly smaller than the footprint of the fridge: it’s very sensitive to any movement (further damage could be caused) and is definitely **not safe** to be plugged in at the moment.

### Progress so far

- Fridge was emptied, its contents were either disposed of (significantly out of date) or placed into a tray next to where some other drinks supplies were located in the Events Space. These are all now covered with the black drape that was previously covering both them and the fridge.
- Back panel was removed. All areas accessible as a result were dusted down, vacuumed and cleaned. 
- No visible faults at this point.
- Attempt at starting fridge was made.
- Fridge started up correctly and, seemingly, began to work, without tripping any fuses.
- I believe the fridge compressor attempted to start, and I detected heat/smell from that area. I immediately turned the fridge off.
- Further disassembly of the fridge undertaken. Bottom plate of fridge removed from main fridge body (though several of its components are still linked to the main fridge, hence the fragility warning above). This plate hosts the power, compressor, radiator and - some of - the control circuitry for the fridge.
- More invasive vacuuming and cleaning of the bottom plate and its components.
- Believing the compressor to be at fault, and with necessary access made possible by removal of the bottom plate, started to disassemble outer components of the compressor (without affecting seals / pressurisation).
- This enabled the following to be identified:
![Exposed compressor wiring by chris-does](https://i.imgur.com/qmnrJl4.jpg)
- As can be seen, the starter relay for the compressor appears to Be damaged.
- Removed the relay from the compressor to allow further inspection:
![Damaged compressor relay by chris-does](https://i.imgur.com/YZICB8F.jpg)
![Damaged relay top by chris-does](https://i.imgur.com/8g52XvQ.jpg)
![Damaged relay connections by chris-does](https://i.imgur.com/27R8wYN.jpg)
![Damaged relay alternative view by chris-does](https://i.imgur.com/UVUzS8s.jpg)
- Based on this inspection, and confirmed by very rudimentary testing, the relay was confirmed as defective.

I don’t believe the level of damage visible could have been caused by my quick on/off test of the fridge as part of this investigation and so I suspect that the original fault/failure caused most of the damage, albeit my test today will have led to some additional impact.

At this point I am not certain if any other aspects of the fridge - in particular the compressor itself - are damaged or defective.

However, I believe it is at least worth obtaining and fitting a replacement relay to remove that as an issue and continue further testing/investigation.

Alas by the time I got this far it was after 18:00 and so, it being a Sunday, there was nowhere open for me to obtain a replacement relay and continue working.

After confirming that there was an external event taking place in the Events Space tomorrow, this is what led me to move the fridge out of the Events Space and to the corner of Room 29 where I often sit so that I may continue working on it tomorrow without interrupting the event attendees.

To maintain ready access to the compressor etc. I have not fully reassembled the fridge and so this is why it is in a fragile state at present.

### Plan for next steps

1. [ ] Identify a suitable replacement relay and acquire it.
2. [ ] Install and test replacement relay on the compressor.
3. [ ] Continue to test/investigate operation of fridge and determine if any other faults exist.
4. [ ] If further faults found, iterate through the test/diagnose/repair/test cycle wherever possible - early warning that if the compressor itself requires replacement (and the fridge is >5 years old) the general recommendation (gleaned from several refrigerator repair forums) appears to be that it is more cost effective to acquire a replacement fridge (which need not necessarily be new of course, just working).
5. [ ] Once no other faults are found, or if deemed not repairable (at least not in line with above recommendations), reassemble fridge and thoroughly clean inside (notably de-mould).
6. [ ] Return fridge to original location in Events Space marked as either operational or awaiting decision regarding replacement/repair beyond above recommendations.

If anyone has any input/suggestions/recommendations or would rather I take a different approach than above, please do let me know.

### Immediate next action

1. [ ] Identify a suitable replacement relay and acquire it.




<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by chris-does** [comment]
A potentially suitable replacement [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) has been identified (awaiting further confirmation with supplier).

This part costs £43.54 exc. VAT and has a lead time of 2-5 days.

However, further reading overnight suggests that - at best - replacing the relay has a 40/60 chance of restoring functionality to the fridge (compressor).

In essence, the fridge repair literature outlines that the most likely reason (60% or more) for the relay to burn out is that the compressor itself has failed, and in this scenario a full compressor replacement or - in the event of the fridge being >5 years old per above - full fridge replacement is recommended.

I will continue to investigate the likelihood of successful repair further and also complete the verification that the identified [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) is indeed the correct part.

However, before proceeding to acquire the part, I would like to gather your - @johnmckerrell, @amcewen  and @DoESsean (and others of course) - input regarding your preferred route forward:

1. Continue with plan outlined in previous post i.e. acquire [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) and attempt further diagnosis of fridge. Acknowledging risk that if compressor is indeed defective, the new relay will fail (if not immediately, then imminently) and the cost of the relay (£43.54 exc. VAT) will be lost / in addition to any replacement.
2. Do not order replacement relay at this time and commence search for suitable replacement fridge to obtain an idea of this cost vs. the cost of the relay.
3. Engage services of refrigeration repair professional to perform analysis of fridge and propose (considerably better informed than I) recommendations.

### Immediate next actions
[ ] Confirm identification of suitable replacement relay.
[ ] Await input from @johnmckerrell, @amcewen & @DoESsean.    

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by johnmckerrell** [comment]
Offline update appears to be that @chris-does was sure that it was just the relay and so has replaced it.

I've turned this on in the last ten minutes, I've plugged it into a sonoff that can be found at http://10.0.36.244 so if it stops working we should be able to track when. "Somebody should" set up a node red flow to alert people when/if that happens.

**Comment by goatchurchprime** [comment]
![image](https://user-images.githubusercontent.com/677254/68877093-689f8a00-06fd-11ea-8cff-0f67e99748c3.png)

**Comment by chris-does** [comment]
Thanks to @amcewen, @goatchurchprime, @magman2112 and @jackie1050  for help and support along the way and to @johnmckerrell and @goatchurchprime for the updates above.

### Overview 
After reading several more refrigerator service manuals, and carrying out some more testing on the events room fridge, I came to the conclusion that it was worth purchasing the identified [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) to determine if the events room fridge could be brought back to life. 
This [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) has now been installed successfully and the fridge is back in the Events Room (fully "de-moulded", cleaned and aired) ready for use.

### Repair report
- After reading the refrigeration literature that suggested a 40/60 chance of success, and then receiving confirmation from the supplier [A S Catering Supplies Ltd.](https://www.ascateringsupplies.com/) that the correct replacement [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) would cost £48.00 exc. VAT and delivery would be a further £8.95 exc. VAT, I felt the risk of failure did not justify the expense.
- With some input from @amcewen, @goatchurchprime and @magman2112 who collectively provided an isolation tester, SONOFF device and some much appreciated wisdom; it was decided to attempt "dummying" the relay i.e. feeding the compressor the electrical supply it would receive if a working relay were connected. After a few attempts, this proved unsuccessful at starting the compressor and so we agreed upon seeking the input of someone with refrigeration expertise (hopefully/perhaps from within the wider DoES community).
- Meanwhile . . . I started investigating the costs of both replacement compressors (new) and also replacement refrigerators (new and pre-loved). It seemed that a complete, fully working, relatively young, and well cared for, second hand refrigerator (many of which were available with a warranty on parts and labour for varying durations) would represent a better investment (<£500) than a new compressor (c£300) and concurrently that a completely new refrigerator (£ _scary_ ) was a significant expense likely not desirable at the best of times, and certainly less than ideal at present.
- This led me to become rather disappointed that I had not been able to be complete this attempted repair, which if successful could seemingly avoid a sizeable outlay.
- Thus began a step-up in my reading of refrigerator service manuals and troubleshooting guides.
- After this, I began to believe that if a few more tests were carried out on the events roomfridge (and that the events room fridge passed of course!) then the probability of fitting a new [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) being sufficient to restore functionality to the events room fridge would reach 70/30 in favour. At this level of risk, I was happy to purchase the [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) in the hope of effecting a successful repair.
- On this basis, I ordered a new [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) from [A S Catering Supplies Ltd.](https://www.ascateringsupplies.com/) and eagerly awaited its arrival (alas the part had to be ordered in and so around a week was lost to the supply chain logistics). Though I must say, [A S Catering Supplies Ltd.](https://www.ascateringsupplies.com/) themselves were most helpful in general and also kept me updated on estimated delivery day. 
- Once the [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) arrived, I began the process of fitting and testing it on the morning of Saturday 9th November. 
- After initial installation, some basic testing showed the fridge to be working in accordance with its specifications.
- The rest of the day was spent demoulding, cleaning and airing the fridge.
- The demoulding process has safely removed all the mould from the fridge, though as a side effect there is some staining to the lining in certain areas (now white rather than silver). I did try to minimise this, but felt that ensuring all the mould was gone trumped aesthetics. I don't believe the variance in colour to be off-putting, and if the fridge is stocked (and/or the internal fridge light on) it is not very visible.
- The cleaning of the interior and exterior of the fridge was completed using [HG hygienic fridge cleaner](https://uk.hg.eu/products/kitchen/hg-hygienic-fridge-cleaner-05l), the remainder of this bottle has been placed in the "Cleaning Products" cupboard under the left hand sink in the kitchen area.
- The cleaning of the interior and exterior glass of the fridge doors was completed using [Windolene](http://rbeuroinfo.com/index.php?SKU=16240&action=product_details.php&COUNTRY=US&LANGUAGE=EN).
- The fridge was then left to air with its fan running, the doors open and the cooling set to "off" so as not to fruitlessly attempt (and hence waste lots of energy) cooling the interior of the fridge.
- The fridge cabinet was reattached to the base unit (which contains the relay, compressor, radiator and other electronics).
- The new [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html) was then installed permanently with new wiring (the old wiring was either burnt as a result of the original relay's failure or too short given the design of the new [relay](https://www.ascateringsupplies.com/start-relay-prodis-nt2.html)).
- The compressor shroud and relay cover were re-installed.
- The protective grill was re-affixed to the back of the base unit of the fridge.
- The fridge shelves were washed and re-installed into the main fridge cabinet.
- Finally the fridge was repositioned back against the wall in the events room and turned on to check it was still working - which it was :-)

### Ongoing usage
During my reading of service manuals and repair guides etc. I encountered several tips and tricks to prolong the life / care for the components of the fridge.

- [ ] I plan to add a page on the Wiki for a "Fridge User Guide" - it seems perhaps a tad excessive, but if avoids (or at least meaningfully delays) incurring additional costs, it is perhaps worthwhile.

- [ ]  @jackie1050 also recommended I print this / create a version of it to be attached to the fridge itself, for those who may not read the Wiki, which I also intend to do.

In advance of the above, the top tips are:
- Do keep the fridge clean (inside and out, including radiator fins/grills and fans etc.)
- Do set the fridge to 2 degrees C.
- Do remove items that will spoil from the fridge before turning the fridge off.
- Don't turn the fridge off while the compressor is running. If you need to turn the fridge off, wait for the compressor to stop first and then turn it off.
- Don't keep the fridge turned on when it is minimally populated and the items therein will not spoil; only turn the fridge on sufficiently in advance of when the items are needed to be cold.
- Don't turn the fridge on and off repeatedly for short periods of time.

**Comment by johnmckerrell** [comment]
@chris-does thanks so much for the update and all your efforts along the way! It's interesting that you say that it should be turned off when not really in use. I wonder if we could leave a SONOFF connected (although that may be overkill) and have it scheduled to come on every evening at 6pm, people in for daytime events could still choose to turn it on by pressing the button on the SONOFF but the schedule would still turn it off again at the end of the day. Do we have any idea how long it would take to chill all the contents and whether that would take more energy than just leaving it on all day?

**Comment by amcewen** [comment]
The fridge has been working fine in the intervening five-and-a-half years, was working today when I checked it.
